src/telemetry/telemetryService.ts

```diff
@@ -189,4 +191,12 @@ export class TelemetryService {
       new ParticipantResponseFailedTelemetryEvent(command, errorName, errorCode)
     );
   }
+
+  trackTreeViewActivated: () => void = throttle(
```

Comment:
Why to we want to throttle here?

Reply:
Due to the way the extension works, there's no integration point we can use to be notified that the side panel is opened, which is why we're piggy-backing on the tree visibility event. However, since we have 3 trees in the side panel (connections, playgrounds, help), that event would get fired 3 times every time we open the sidepanel. Rather than deal with this in post-processing, I figured throttling would be a reasonable way to reduce the noise.